src/mokka/src/main/java/pl/hycom/mokka/emulator/mock/handler/impl/GroovyMockHandler.java

```diff
@@ -40,8 +41,7 @@ public void handle(MockConfiguration mockConfiguration, MockContext ctx) throws
 			Binding binding = new Binding();
 
 			binding.setVariable("ctx", ctx);
-			binding.setVariable("mockConfiguration", mockConfiguration);
-			binding.setVariable("httpHelper", httpHelper);
```

Comment:
`httpHelper` field may be removed and if there is no other usages of `HttpHelper` class -> remove it also.
I believe you should also update the implementation of `createResponseLog`


Would you also consider writing unit test?